txros/src/txros/util.py

```diff
@@ -117,17 +132,15 @@ def _step(cur, gen, currently_waiting_on, mine, df, has_been_cancelled):
                 ultimateTrace = appCodeTrace
                 while ultimateTrace.tb_next.tb_next:
                     ultimateTrace = ultimateTrace.tb_next
-                filename = ultimateTrace.tb_frame.f_code.co_filename
-                lineno = ultimateTrace.tb_lineno
```

Comment:
The `filename` and `lineno` variables were not used, so their assignments were removed.